src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,91 @@
+import model from '../db/models';
+
+const { Bookmark, Articles } = model;
+/**
+ * Bookmark class
+ */
+class BookmarkManager {
+  /**
+     *
+     * @param {Object} req
+     * @param {Object} res
+     * @return {Object} bookmarked story
+     */
+  static async create(req, res) {
+    try {
+      const { id } = req.user;
+      const { slug } = req.params;
+
+      const findArticle = await Articles.findOne({ where: { slug } });
+      const createBookmark = await Bookmark.create({
+        userId: id,
+        slug,
+        title: findArticle.dataValues.title,
+        description: findArticle.dataValues.description
+      });
+      return res.status(201).json({
+        message: 'You have bookmarked successfully',
+        data: createBookmark
+      });
+    } catch (error) {
+      return res.status(500).json({
```

Comment:
this statement does not check the article has already been bookmarked

Reply:
I think the only way you can that response is when you try to post the same article because there is a restriction of that in the database